src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/ActivityPanel.kt

```diff
@@ -0,0 +1,51 @@
+package org.wycliffeassociates.otter.jvm.app.widgets
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.scene.Node
+import javafx.scene.layout.HBox
+import javafx.scene.layout.Priority
+import javafx.scene.paint.Color
+import org.wycliffeassociates.otter.jvm.app.UIColorsObject.Colors
+import org.wycliffeassociates.otter.jvm.app.widgets.WidgetsStyles.Companion.activityPanelButton
+import tornadofx.*
+import tornadofx.Stylesheet.Companion.root
+
+class ActivityPanel(graphicLeft: Node, buttonLeftColor: Color,
+                    graphicMiddleLeft: Node, buttonMidLeftColor: Color,
+                    graphicMiddleRight: Node, buttonMidRightColor: Color,
+                    graphicRight: Node, buttonRightColor: Color) : HBox() {
+
```

Comment:
Could we change this to have a list of Nodes and corresponding colors to make it a more general component? That way if we needed to add/remove a graphic later we wouldn't need to change this class

Reply:
yes that'd be good